### PR TITLE
Dashboard styling fixes

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch7618]

**Comment by FrederikBolding** [comment]
DONT MERGE - Pending fixes to svgs

File: src/theme/theme.ts [hunk 1]
```diff
@@ -104,7 +104,10 @@ const TEXT_VARIANTS = {
   link: {
     fontSize: { _: 0, sm: 1 },
     lineHeight: { _: 0, sm: 1 },
-    color: 'BLUE_BRIGHT'
+    color: 'BLUE_BRIGHT',
+    '&:hover': {
+      color: '#005166'
+    }
```
Comment (GuillaumeRx) [inline src/theme/theme.ts:110]: is this working ???

**Review by GuillaumeRx — APPROVED** [review]

File: src/theme/theme.ts (hunk 1 above)
Comment (FrederikBolding) [reply src/theme/theme.ts:110]: Yes, you can see it by hovering over the new TokenPanel buttons 😄

**Pull request merged**